ethereum/sidecar/server.go

```diff
@@ -193,6 +190,13 @@ func RunServer(
 		panic(fmt.Sprintf("failed to connect to the Ethereum network: %v", err))
 	}
 
+	accountAddress := chain.Key().Address
+
+	logger.Info(
+		"Extracted Ethereum account address",
+		"ethereum_account_address", accountAddress,
+	)
```

Comment:
We already print the address, see the "successfully extracted private key from keyring" log.

Reply:
Removed the log in a386fefb83571a5a2a457cb92fd5fc44896c13c7.